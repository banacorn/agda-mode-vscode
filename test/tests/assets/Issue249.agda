-- https://github.com/banacorn/agda-mode-vscode/issues/249
module Issue249 where

open import Agda.Builtin.Sigma
open import Agda.Builtin.Unit

bug : Σ ⊤ λ _ → ⊤
bug = ?

{- This is a comment. -}