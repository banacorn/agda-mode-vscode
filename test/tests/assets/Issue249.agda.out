-- https://github.com/banacorn/agda-mode-vscode/issues/249
module Issue249 where

open import Agda.Builtin.Sigma
open import Agda.Builtin.Unit

bug : Σ ⊤ λ _ → ⊤
bug = tt , {!   !}

{- This is a comment. -}